features/moviedetail/src/main/java/com/tkpd/moviedetail/view/MovieDetailFragment.kt

```diff
@@ -47,53 +60,83 @@ class MovieDetailFragment : Fragment() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         arguments?.let {
-            movieId = it.getInt(MovieConstant.PARAM_MOVIE_ID, 0)
+            movieId = it.getInt(PARAM_MOVIE_ID, 0)
         }
     }
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
-    ): View? {
-        // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_movie_detail, container, false)
+    ): View {
+        return ComposeView(requireContext()).apply {
+            setContent {
+                MovieDetailMainLayout(viewModel)
+            }
+        }
     }
 
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        showLoading()
-        getErrorLayout.hide()
         viewModel.getMovieList(movieId ?: 0)
```

Comment:
will be good, `viewModel.getMovieList(movieId ?: 0)` can move to view model init